src/components/interaction-designer/Block.vue

```diff
@@ -373,33 +365,7 @@ export class Block extends mixins(Lang) {
     return data?.targetId === block.uuid
   }
 
-  updateTranslatedBlockEditorPosition(): void {
```

Comment:
no longer needed because we 're having a fixed position at the right of the screen

Reply:
I'm so glad this timer magic is gone 🎉 thanks!

Reply:
so do I 😄